jennite-vcf-store/src/main/dist/deb/debian/postinst

```diff
@@ -14,10 +14,19 @@ case "$1" in
         . /etc/default/opal
         mkdir -p $OPAL_HOME/plugins
         if [ -d "$OPAL_HOME"/plugins ]; then
+        	OLD_PLUGIN=$(ls -t "$OPAL_HOME"/plugins/ | grep jennite-vcf-store | head -1)
+
           cp -r /usr/share/jennite* $OPAL_HOME/plugins
+
+          if [ ! -z "$OLD_PLUGIN" ]; then
```

Comment:
Maybe you could check for site.properties in $OLD_PLUGIN as this is the file you will copy?